bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/niri/Niri.java

```diff
@@ -797,20 +793,6 @@ public ROIDescription(ROIDescription roid) {
         }
 
         /**
-         * Return the x start pixel.
-         */
-        public int getXStart() {
-            return _xStart;
-        }
-
-        /**
-         * Return the y start pixel.
-         */
-        public int getYStart() {
-            return _yStart;
-        }
-
-        /**
```

Comment:
Remove. If it was used in ITC you would immediately notice it when doing a full recompile. The only dependency LCH/LTTS has is the XML interface. And I think the usage of reflection is limited to the initialization of the UI components in the OT.